dulwich/porcelain.py

```diff
@@ -1753,6 +1755,76 @@ def update_head(repo, target, detached=False, new_branch=None):
             r.refs.set_symbolic_ref(b"HEAD", to_set)
 
 
+def reset_file(repo, file_path: str , target: bytes = b'HEAD'):
+    """
+    reset the file to specific commit or branch
```

Comment:
Per google doc string style and the rest of Dulwich, please Put the summary immediately after the triple quotes, capitalize and end with a trailing period.